docs/build/isc/v1.0.0-rc.6/docs/_admonitions/_off-ledger-requests.md

```diff
@@ -0,0 +1,5 @@
+:::note
+
+If you are using EVM, you will only need to use off-ledger requests for most of your contracts.
```

Comment:
only and most seem to communicate different things 😅 
But I would rephrase this in general. As you can't really use on-ledger requests for anything outside magic or core contracts